packages/nube-devtools/src/contentScript/index.ts

```
@@ -0,0 +1,33 @@
+})
+
+window.addEventListener('DOMContentLoaded', () => {
+  chrome.runtime.sendMessage({ action: 'injectNubeSDKEventListeners' }, () => {
+    if (chrome.runtime.lastError) {
+      console.error('Error sending injectNubeSDKEventListeners message:', chrome.runtime.lastError);
+    }
+  });
+})
+
+window.addEventListener('NubeSDKEvents', (event: Event) => {
+  const data = JSON.stringify((event as any).detail)
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)